### PR TITLE
add a function to draw mean +/- std.dev.

### PR DESCRIPTION
**Pull request by dsekihat**
add a functin to draw mean +/- std.dev.

### CODE REVIEW

File: tpcwithdnn/dnn_optimiser.py [hunk 1]
```diff
@@ -442,23 +441,55 @@ def draw_multievent_hist(self, events_counts, func_label, hist_name, source_hist
...
+                        hist.SetFillColor(colors[i])
+                        hist.SetFillStyle(3001)
+                        hist.Draw("sameE2")
+                        # train_events_k = train_events / 1000
+                        leg.AddEntry(hist, "N_{ev}^{training} = %d" % train_events, "FP")
+
                     root_file.Close()
 
                 leg.Draw()
+                #canvas.Modified();
```
Comment (saganatt) [inline tpcwithdnn/dnn_optimiser.py:477]: Could you remove these comments?

Comment (saganatt) [reply tpcwithdnn/dnn_optimiser.py:477]: Or are they used for something?

Comment (dsekihat) [reply tpcwithdnn/dnn_optimiser.py]: Yes, done.

File: tpcwithdnn/dnn_optimiser.py [hunk 2]
```diff
@@ -13,10 +13,10 @@
 from tensorflow.keras.utils import plot_model
 
 from root_numpy import fill_hist # pylint: disable=import-error
-
 from ROOT import TH1F, TH2F, TFile, TCanvas, TLegend, TPaveText, gPad # pylint: disable=import-error, no-name-in-module
 from ROOT import gStyle, kWhite, kBlue, kGreen, kRed, kCyan, kOrange, kMagenta # pylint: disable=import-error, no-name-in-module
 from ROOT import gROOT  # pylint: disable=import-error, no-name-in-module
+#from ROOT import TPython  # pylint: disable=import-error, no-name-in-module
```
Comment (saganatt) [inline tpcwithdnn/dnn_optimiser.py:19]: Could you remove this?

File: tpcwithdnn/dnn_optimiser.py [hunk 3]
```diff
@@ -442,23 +441,52 @@ def draw_multievent_hist(self, events_counts, func_label, hist_name, source_hist
...
+                        hist.SetFillStyle(3001)
+                        hist.Draw("sameE2")
+                        # train_events_k = train_events / 1000
+                        leg.AddEntry(hist, "N_{ev}^{training} = %d" % train_events, "FP")
+
                     root_file.Close()
 
                 leg.Draw()
                 self.add_desc_to_canvas()
                 self.save_canvas(canvas, frame, "plots/20200803", hist_name, file_formats)
```
Comment (saganatt) [inline tpcwithdnn/dnn_optimiser.py:478]: We can update the prefix to something like 20201210. Or what about "plots/perf"?

**Review by saganatt — APPROVED** [review]
I have just two more nitpicks after trying it.

File: tpcwithdnn/dnn_optimiser.py (hunk 2 above)
Comment (dsekihat) [reply tpcwithdnn/dnn_optimiser.py]: done

File: tpcwithdnn/dnn_optimiser.py (hunk 3 above)
Comment (dsekihat) [reply tpcwithdnn/dnn_optimiser.py]: datetime is added, instead of hardcoded date. the date will be automatically "today".

**Comment by saganatt** [comment]
Could you fix new pylint errors?

**Pull request merged**